core/src/main/java/org/elasticsearch/index/indexing/IndexingStats.java

```diff
@@ -114,32 +112,21 @@ public long getDeleteCount() {
         /**
          * Returns if the index is under merge throttling control
          */
-        public boolean isThrottled() {
-            return isThrottled;
-        }
+        public boolean isThrottled() { return isThrottled; }
 
         /**
-         * Gets the amount of time in milliseconds that the index has been under merge throttling control
+         * Gets the amount of time in a TimeValue that the index has been under merge throttling control
```

Comment:
You don't even really need to say "a TimeValue" here because its in the return type. Its only important when its a long because you don't know if its millis or nanos.